### PR TITLE
Split the class and feat prerequisite ignore into two separate settin…

### PR DESCRIPTION
**Pull request by nihilistzsche**
…gs so that classes can be unlocked but feats still filtered.  #605.

### CODE REVIEW

File: ToyBox/ReadMe.md [hunk 1]
```diff
@@ -38,30 +38,23 @@ WARNING: this tool can both miraculously fix your broken progression or it can b
...
-      * Fixed issue that generated bogus extra answers for non conditional response by ensuring that the feature only gets activated by a set of continuation cues that all have conditions
-    * **Remote Companion Dialog**
-     	* (***Ronin***) Convinced Daeran to stop repeating himself in the Grey Garrison
+  * **Expand Answers For Conditional Responses**
+    * Dialog dependent on previous dialog checks now works correctly with the feature enabled
+    * Fixed issue that generated bogus extra answers for non conditional response by ensuring that the feature only gets activated by a set of continuation cues that all have conditions
+  * **Remote Companion Dialog**
+   	* (***Ronin***) Convinced Daeran to stop repeating himself in the Grey Garrison
   * **Camera**
-    * Mouse3 Camera + forward/back keys now lets you fly freely when pitch is enabled
```
Comment (cabarius) [inline ToyBox/ReadMe.md:57]: Why is this being removed?

**Comment by cabarius** [comment]
It looks like this was not properly rebased with master.  Please rebase so we don't lose more recent updates.

File: ToyBox/classes/Models/Settings.cs [hunk 2]
```diff
@@ -118,8 +118,6 @@ public class Settings : UnityModManager.ModSettings {
         public bool toggleRotateOnAllMaps = false;
         public bool toggleScrollOnAllMaps = false;
         public bool toggleCameraPitch = false;
-        public bool toggleCameraElevation = false;
```
Comment (cabarius) [inline ToyBox/classes/Models/Settings.cs:121]: these changes should not be deleted.  Please rebase and and apply your patch to top of tree.

**Review by cabarius — CHANGES_REQUESTED** [review]
Please rebase against master so we don't lose my more recent changes.

**Comment by nihilistzsche** [comment]
Hi, I tried to rebase it to master but when I go to rebase it appears in my history to already be on top of current master.   Is there another change missing from the repo?

**Review by cabarius — APPROVED** [review]

**Comment by cabarius** [comment]
I merged this and it turned out to be wrong.  I am reverting this patch.  Please check out master and apply just the changes you made to top of tree.

**Pull request merged**